ocaml/xapi/xapi_globs.ml

```diff
@@ -1023,6 +1023,8 @@ let python3_path = ref "/usr/bin/python3"
 let observer_experimental_components =
   ref (StringSet.singleton Constants.observer_component_smapi)
 
+let pool_recommendations_dir = ref "/etc/xapi.pool-recommendations.d"
```

Comment:
We are currently reading all files in this directory. I would think about only reading files that have an extension like '*.rec' and filter out other ones. It's just good practice to use extensions on files because it also helps editors to recognise them.